docs/grammar.txt

```diff
@@ -142,9 +142,11 @@ Uint = 'uint' | 'uint8' | 'uint16' | 'uint24' | 'uint32' | 'uint40' | 'uint48' |
+FixedSize = ( '8' | '16' | '24' | '32' | '40' | '48' | '56' | '64' | '72' | '80' | '88' | '96' | '104' | '112' | '120' | '128' | '136' | '144' | '152' | '160' | '168' | '176' | '184' | '192' | '200' | '208' | '216' | '232' | '240' | '248' | '256' )
```

Comment:
I am not fully sure it makes sense restricting the types on the grammar level, it just results in a giant list and certain combinations are currently ruled out in type checking.

Parser only checks that `W` must be between 8 and 256 and divisible by 8, but `F` is only checked for <= 80.

We could decide to make all `W`/`F` valid are raise an error in the type checker (same applies to `int`, `uint`, `bytes`)